### ISSUE TITLE
Saving some sent attachments to storage results in file saved with 0B size

### ISSUE THREAD

**Issue by DirkPitt1** [signalapp/Signal-Android#10119]
- [x] I have searched open and closed issues for duplicates
- [x] I am submitting a bug report for existing functionality that does not work as intended
- [x] I have read https://github.com/signalapp/Signal-Android/wiki/Submitting-useful-bug-reports
- [x] This isn't a feature request or a discussion topic

----------------------------------------

### Bug description
Saving some sent attachments (pdf and doc files) to storage results in file saved with 0B size.

### Steps to reproduce
- receive/send some pdf file
- save it to storage

**Actual result:** Saved file has 0B size.
**Expected result:** File is saved correctly.

### Screenshots

![Attach1](https://user-images.githubusercontent.com/28538453/96893913-bcbcfd80-147a-11eb-9073-37dbcd6a9d02.jpg)

![Attach2](https://user-images.githubusercontent.com/28538453/96893935-c2b2de80-147a-11eb-804c-bea5ccde1cdd.jpg)

### Device info

**Device:** LG G4
**Android version:** 6.0
**Signal version:** 4.74.3

### Link to debug log

https://debuglogs.org/ab19f549cb455c3cd5fe2e938acac50f6ff70cf3beb06e45e1a1d51cc8b0ac7c

from other device
https://debuglogs.org/dad131df9454bed0ec4cc64dbc0c4c420f81c783ddbd74ebb71fec70772deefd

**Comment by cody-signal** [comment]
Hi @DirkPitt1, I'm unable to repro this. Does this happen every time with any attachment or just happened randomly? Also which of the logs was the sending device, which the receiving, and which of those two was the save that failed?

**Comment by DirkPitt1** [comment]
Hi @cody-signal , I've sent some pdf and doc files to my contact who couldn't save them correctly. Then I tried to save these files from message on my phone and also had files with 0B size. It was the first time when I encountered this problem but some of my contacts said they encountered something like that once or twice before.

The first log is my sending device, the second is my contact's receiving device. The save failed on both.

I tried some more, sending some files to Note to self and I confirmed following facts:
- file type seems to be irrelevant, doc, pdf, epub, mobi, all are saved with 0B size
- images seem to be saved correctly
- revoking and allowing Signal permission to storage didn't fix the issue
- I've got nearly 10GB free space on internal memory so it's not lack of space
- It doesn't matter if the file was originally sent from Android or Desktop


Here's my new debug log after the tries with Note to self:
https://debuglogs.org/6c457a07b9ad8040b384200c2ea09f5f8e1150ef62d6d8fcdd9be316f82b2563
